docs/help-topics/stonesoup-whatsnext/stonesoup-whatsnext.yml

```diff
@@ -4,14 +4,16 @@
 
 - name: stonesoup-whatsnext-add-component 
   tags:
-  title: Add a component 
+  title: Add more components 
   content: |-
-    To extend your application, add source code to your repository. Stonesoup then builds a component or image from this source code. 
+    To extend your application, add source code to your repository. We'll build a component or image from this source code. 
```

Comment:
`We'll build a component or image from this source code.`

Removal of stonesoup is good here. However, it's confusing to say that `a component or image` will be built. A component represents an abstraction that is backed by a git source code repository (or container image but this feature was removed a while back). We also use this description in the app `A “component” is an image that we build from source code in a repository`

Therefore I think it's fair to simply say `We'll build an image from this source code. `

Reply:
@christianvogt Thank you for voicing my exact concern about this sentence, I shared it in https://issues.redhat.com/browse/RHTAPBUGS-303 but it was decided to only remove 'stonesoup' for now and address content issues later. So I reverted my suggestions from the 1st commit here.

Nevertheless I'll add your wording since you're an SME who points to a confusing place. Thanks.